src/components/memory/view/memory.html

```diff
@@ -0,0 +1,27 @@
+<div class="container-fluid">
+  <div class="row">
+    <div class="panel panel-default">
+      <div class="panel-heading">
+        <h3 class="panel-title">Memory</h3>
+      </div>
+      <div class="panel-body">
+        <div class="form-horizontal" ng-if="!$ctrl.isMemoryDefined()">
+          <div class="form-group">
+            <label class="col-md-8 control-label">Memory management has not been defined. Click
+              <button ng-click = "$ctrl.createDefaultMemory()"> here</button> to define it</label>
```

Comment:
Ok, makes sense! Do you prefer no option instead of this message? I am breaking UI consistency with this message.